### PR TITLE
Merged host and SDK help

### PR DESCRIPTION
**Pull request by wli3**
Change help according to https://github.com/dotnet/cli/issues/6215

### CODE REVIEW

**Comment by wli3** [comment]
@ramarag I added some _._ that is different from your comment

File: test/dotnet-help.Tests/GivenThatIWantToShowHelpForDotnetCommand.cs [hunk 1]
```diff
@@ -15,44 +15,46 @@ namespace Microsoft.DotNet.Help.Tests
...
 
-Arguments:
-  [command]             The command to execute
-  [arguments]           Arguments to pass to the command
-  [host-options]        Options specific to dotnet (host)
-  [common-options]      Options common to all commands
+path-to-application:
+  The path to an application .dll file to execute.
+
+SDK commands:
```
Comment (wli3) [inline test/dotnet-help.Tests/GivenThatIWantToShowHelpForDotnetCommand.cs]: This text from test is more clear about what is end result

File: src/dotnet/commands/dotnet-help/LocalizableStrings.cs [hunk 2]
```diff
@@ -79,7 +79,26 @@ internal class LocalizableStrings
 
         public const string CommandArgumentDescription = "CLI command for which to view more detailed help.";
 
+        public const string PathToApplicationDefinition = "The path to an application .dll file to execute.";
 
+        public const string SDKVersionCommadDefination = "Display .NET Core SDK version.";
```
Comment (nguerrera) [inline src/dotnet/commands/dotnet-help/LocalizableStrings.cs]: Typo: Definition

**Comment by livarcocc** [comment]
@wli3 please, fill up the template like here: https://github.com/dotnet/cli/pull/6627.

File: src/dotnet/commands/dotnet-help/LocalizableStrings.cs [hunk 3]
```diff
@@ -79,7 +79,26 @@ internal class LocalizableStrings
 
         public const string CommandArgumentDescription = "CLI command for which to view more detailed help.";
 
+        public const string PathToApplicationDefinition = "The path to an application .dll file to execute.";
```
Comment (livarcocc) [inline src/dotnet/commands/dotnet-help/LocalizableStrings.cs]: Should this be path to an <application>.dll to match the pattern below?

**Comment by wli3** [comment]
@MattGertz 
Could you review this change?

**Customer scenario**

User type dotnet -h to look for help in command line

quote Richard

_Here is my proposal for the new dotnet -h (and --help). The top-level point is merging help content from the host and the sdk. The actual text may change after this change, but the mechanism will not and also won't require coordinate between the two sides. The only shared bit of content is really the the usage lines at the top._

**Bugs this fixes**

https://github.com/dotnet/cli/issues/6215

**Workarounds, if any**

N/A

**Risk**

The user will be confused by cannot get SDK help when invoke help command

**Performance impact**

no

**Root cause analysis**

N/A

**How was the bug found?**

N/A

File: src/dotnet/commands/dotnet-help/LocalizableStrings.cs (hunk 3 above)
Comment (wli3) [inline src/dotnet/commands/dotnet-help/LocalizableStrings.cs]: The pattern is like PascalCase the "key" + defination. So the "key" is 
```
path-to-application:
```
So I made it this way

File: src/dotnet/commands/dotnet-help/LocalizableStrings.cs (hunk 2 above)
Comment (wli3) [inline src/dotnet/commands/dotnet-help/LocalizableStrings.cs]: Fixed every Defination

**Pull request merged**